### PR TITLE
Update gcp discovery docs with access scopes

### PR DESCRIPTION
**Pull request by atburke**
This change updates the GCP auto-discovery docs to include a step for expanding the access scopes on an instance. It also updates the custom role screenshot to correctly show all permissions needed.

### CODE REVIEW

File: docs/pages/enroll-resources/auto-discovery/servers/gcp-discovery.mdx [hunk 1]
```diff
@@ -93,6 +93,12 @@ discover instances.
   your custom role. Click *Save*.
 
   ![Role assignment](../../../../img/server-access/guides/gcp/assign-role-to-service-account@2x.png)
+
+  If the Discovery Services will run in a GCP compute instance, edit the instance and
```
Comment (ptgott) [inline docs/pages/enroll-resources/auto-discovery/servers/gcp-discovery.mdx:97]:
```suggestion
  If the Discovery Service will run in a GCP compute instance, edit the instance and
```

File: docs/pages/enroll-resources/auto-discovery/servers/gcp-discovery.mdx [hunk 2]
```diff
@@ -93,6 +93,12 @@ discover instances.
...
+  assign the service account to the instance and set its Access Scopes to include
```
Comment (ptgott) [inline docs/pages/enroll-resources/auto-discovery/servers/gcp-discovery.mdx:98]: Is "Access Scopes" supposed to be a proper noun? (I.e., is it a GCP-specific feature?) If not, we can use "access scopes".

File: docs/pages/enroll-resources/auto-discovery/servers/gcp-discovery.mdx [hunk 3]
```diff
@@ -131,6 +137,14 @@ discover instances.
   --member="serviceAccount:teleport-discovery@<Var name="project_id" />.iam.gserviceaccount.com" \
   --role="projects/<Var name="project_id" />/roles/teleport_discovery"
   ```
+
+  If the Discovery Services will run in a GCP compute instance, run the following command to
```
Comment (ptgott) [inline docs/pages/enroll-resources/auto-discovery/servers/gcp-discovery.mdx:141]:
```suggestion
  If the Discovery Service will run in a GCP compute instance, run the following command to
```

**Review by ptgott — APPROVED** [review]
Approved with minor grammar/style suggestions

**Review by rosstimothy — APPROVED** [review]

**Pull request merged**